packages/app/obojobo-repository/shared/components/module.scss

```diff
@@ -23,6 +23,10 @@
 		word-wrap: break-word;
 	}
 
+	:focus {
+		outline-offset: 9px;
```

Comment:
`9px` seems like it's kind of a magical number in this situation. Ideally it would match some other number which is causing the focused element to be offset from the edges of its parent in the first place.

There's an earlier rule for `.repository--module-icon` that sets `padding` to `0.8em`. Most likely this is the value you'd need to use. Ideally could set this to a variable and use that variable in both places for consistency.